hc-sr04-range-sensor/Makefile

```diff
@@ -92,6 +100,70 @@ $(ELF1): $(SRC1) $(HEADERS) | $(OUT)
 clean:
 	$(RM) -fr $(ELF0) $(ELF1) $(OUT)
 
+install_PRU0:
+	@echo 'Installation Process Initiated'
+ifeq ($(CURRENTSTATE0),$(STATESRUNNING))
```

Comment:
Use shell commands to achive conditional logic. Build rule lines are not restricted to simple tool invocation. You can use shell's || or && or []